### PR TITLE
[flink] Introduce CreateTagFromTimestampAction for flink

### PR DESCRIPTION
**Pull request by xuzifu666**
### Purpose

Linked issue: close #
Follow up https://github.com/apache/paimon/pull/4051
introduce action for create_tag_from_timestamp.

### CODE REVIEW

File: paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CreateTagFromTimestampActionITTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    private static final RowType ROW_TYPE = RowType.of(FIELD_TYPES, new String[] {"k", "v"});
+
+    @BeforeEach
+    public void setUp() {
+        init(warehouse);
+    }
+
+    @Test
+    public void testExpirePartitionsAction() throws Exception {
```
Comment (LinMingQiang) [inline paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CreateTagFromTimestampActionITTest.java:54]: testExpirePartitionsAction ?

Comment (xuzifu666) [reply paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CreateTagFromTimestampActionITTest.java:54]: It a mistake, I change it. Thanks~ @LinMingQiang

File: paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CreateTagFromTimestampActionITTest.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                        database + "." + tableName,
+                        "--tag",
+                        tag,
+                        "--timestamp",
+                        Long.toString(ts))
+                .run();
+
+        Snapshot snapshot = table.tagManager().tags().firstKey();
+
+        assertThat(table.tagManager().tagExists(tag)).isTrue();
```
Comment (LinMingQiang) [inline paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CreateTagFromTimestampActionITTest.java:80]: You should add more test case  see -.- > CreateTagFromTimestampProcedureITCase.

**Comment by LinMingQiang** [comment]
+1

**Comment by JingsongLi** [comment]
+1

**Pull request merged**